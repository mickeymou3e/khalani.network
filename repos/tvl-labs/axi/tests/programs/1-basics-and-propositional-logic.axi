../../tutorial/1-basics-and-propositional-logic.axi