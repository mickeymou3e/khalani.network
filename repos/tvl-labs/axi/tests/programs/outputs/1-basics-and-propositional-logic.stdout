Result: Nothing
Result: Nothing
Result: Nothing
Result: Nothing
Result: Proved(if(not(not(a)), a))
Result: Proved(if(a, if(b, and(a, b))))
Result: Proved(if(a, if(b, b)))
Result: Proved(if(and(a, b), or(a, c)))
Result: Proved(if(if(or(a, c), d), if(and(a, b), d)))
Result: Proved(if(if(a, b), if(if(b, a), iff(a, b))))
Result: Proved(if(iff(a, b), and(if(a, b), if(b, a))))
